make-version/make-version.py

```diff
@@ -39,6 +53,8 @@ def update_py(version, pathname):
             update_py(version, project_root.joinpath(filepath))
         elif filepath.endswith('.json'):
             update_json(version, project_root.joinpath(filepath))
+        elif filepath.endswith('.yaml'):
+            update_json(version, project_root.joinpath(filepath))
```

Comment:
I guess this should read `update_yaml`. Did you actually try it?

Reply:
I already fixed that once. Probably got left out while switching branches.